src/migrations/20210106160953-create-activity-participants.js

```diff
@@ -0,0 +1,53 @@
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    queryInterface.createTable('ActivityParticipants', {
```

Comment:
Kinda similar naming confusion here. I was thinking of Participants as individual people, or roles at the grantee that were at the activity. The PR is using participants for both that as well as the Grantee(s) or NonGrantee(s)

Reply:
Yeah I agree it is confusing, I just lived with the confusion enough that I forgot about it! I'll ping the other engineers and try to come up with a better name.

Reply:
What do you think about renaming `ActivityParticipants` to `ActivityReceivers` or `ActivityRecipients`?